### PR TITLE
stop generating masked csrf tokens for now

### PR DESCRIPTION
**Pull request by nealharris**
@tamird @gglider

### CODE REVIEW

**Comment by ggilder** [comment]
LGTM except I think you meant @ggilder :wink:

**Comment by ggilder** [comment]
actual test failure tho

**Comment by ggilder** [comment]
```
72) Failure:
test_enum_rjs_test(RenderTest) [/home/travis/build/square/rails/actionpack/test/controller/render_test.rb:1037]:
<"$$(\".product\").each(function(value, index) {\nnew Effect.Highlight(element,{});\nnew Effect.Highlight(value,{});\nSortable.create(value, {onUpdate:function(){new Ajax.Request('/test/order', {asynchronous:true, evalScripts:true, parameters:Sortable.serialize(value) + '&authenticity_token=' + encodeURIComponent('QUFBQQAAAAA=')})}});\nnew Draggable(value, {});\n});"> expected but was
<"$$(\".product\").each(function(value, index) {\nnew Effect.Highlight(element,{});\nnew Effect.Highlight(value,{});\nSortable.create(value, {onUpdate:function(){new Ajax.Request('/test/order', {asynchronous:true, evalScripts:true, parameters:Sortable.serialize(value) + '&authenticity_token=' + encodeURIComponent('QUFBQQ==')})}});\nnew Draggable(value, {});\n});">.
```

**Comment by nealharris** [comment]
Argh, sorry @ggilder !  

And yeah, just pushed up changes to the tests to expect the old behavior.

**Comment by ggilder** [comment]
cool, LG
